common/scripted_triggers/01_scripted_triggers_combat.txt

```diff
@@ -382,7 +382,10 @@ is_allowed_sparring_duels_trigger = {
 	}
 }
 this_can_duel_with_from_trigger = { 
-	NOR = { 
+	NOR = {
+		# Warcraft
+        is_untouchable_trigger = no
```

Comment:
Should be `yes` since placed in `NOR` block.
```suggestion
        is_untouchable_trigger = yes
```

Reply:
This should be resolved in the latest commit.